packages/documentation/package.json

```diff
@@ -12,8 +12,8 @@
     "test": "docusaurus build"
   },
   "dependencies": {
-    "@docusaurus/core": "2.0.0-alpha.65",
-    "@docusaurus/preset-classic": "2.0.0-alpha.65",
+    "@docusaurus/core": "2.0.0-alpha.66",
+    "@docusaurus/preset-classic": "2.0.0-alpha.66",
```

Comment:
There are some issues with this version of Docusaurus which is why @mwcz rolled back this version in his last PR.